website/docs/engine-flink/datastream.md

```diff
@@ -0,0 +1,386 @@
+---
+title: "Datastream API"
+sidebar_position: 6
+---
+
+<!--
+ Copyright (c) 2025 Alibaba Group Holding Ltd.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+# Fluss Datastream
+## Overview
+The Fluss Datastream API provides a Flink DataStream source implementation for reading data from Fluss tables. It allows you to seamlessly integrate Fluss tables with Flink's DataStream API, enabling you to process data from Fluss in your Flink applications.
+
+Key features of the Fluss Datastream API include:
+* Reading from both primary key tables and log tables
+* Support for projection pushdown to select specific fields
+* Flexible offset initialization strategies
+* Custom de/serialization schemas for converting between Fluss records and your data types
+* Automatic handling of updates for primary key tables
+
+## Dependency
+In order to use the Fluss Datastream API, you need to add the following dependency to your `pom.xml` file:
+
+```xml
+<!-- https://mvnrepository.com/artifact/com.alibaba.fluss/fluss-client -->
```

Comment:
This should be `https://mvnrepository.com/artifact/com.alibaba.fluss/fluss-flink-xxx`